packages/my-account/src/components/composite/subscription/SubscriptionPayments.tsx

```diff
@@ -0,0 +1,76 @@
+import { OrderSubscription } from "@commercelayer/sdk"
+import capitalize from "lodash/capitalize"
+import { Trans } from "react-i18next"
+
+import Empty from "#components/composite/Empty"
+
+import { getCardDetails, getCardIconFromDetails } from "#utils/getCardDetails"
+
+interface Props {
+  orderSubscription?: OrderSubscription
+}
+
+function SubscriptionPayments({ orderSubscription }: Props): JSX.Element {
+  if (
+    orderSubscription != null &&
+    orderSubscription.customer_payment_source != null &&
+    orderSubscription.customer_payment_source.payment_method != null &&
+    orderSubscription.customer_payment_source.payment_source != null
+  ) {
+    const paymentMethod =
+      orderSubscription.customer_payment_source.payment_method
+    const paymentSource =
+      orderSubscription.customer_payment_source.payment_source
```

Comment:
I would change this to increase readability.

```suggestion
  const paymentMethod =
    orderSubscription?.customer_payment_source?.payment_method
  const paymentSource =
    orderSubscription?.customer_payment_source?.payment_source

  if (paymentMethod != null && paymentSource != null) {
```

Reply:
Thanks for the suggestion!